frontend/src/components/users/GridTools.ts

```diff
@@ -1,24 +1,24 @@
-import { Footer } from '@mantine/core'
-import { Coordinate, WidgetDimension, Widgets, WidgetType } from 'common'
-import { widget } from 'components/widgets/widgets.stories'
+import { WidgetDimension, Widgets, WidgetType } from 'common'
+
 import { cartesianProduct, range, replicate } from 'utils'
+import { pos, size, Vec2 } from 'vec2'
 
-export const gridSize = { w: 5, h: 3 } as const
+export const gridSize = pos(5, 3)
```

Comment:
```suggestion
export const gridSize = size(5, 3)
```
격자 크기이니 `Pos`가 아닌 `Size`를 사용하는 것이 좋을 것 같습니다.